CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 
 ## [2.5.8] - not released yet
 
+### Added
```

Comment:
Things have already been added to the `CHANGELOG.md`, you may want to rebase your PR